examples/synthetics_mesh_example_test.go

```
@@ -85,15 +86,17 @@ func printMetricsMatrixGRPC(matrix metricsMatrixGRPC) {
 		row := fromAgent + "\t"
 		for _, toAgent := range matrix.agents {
 			if metrics, ok := matrix.getMetricGRPC(fromAgent, toAgent); ok {
-				row = row + formatLatencyGRPC(metrics) + "\t"
+				row += formatLatencyGRPC(metrics) + "\t"
 			} else {
-				row = row + "[X]\t"
+				row += "[X]\t"
 			}
 		}
 		fmt.Fprintln(w, row)
 	}
 
-	w.Flush()
+	if err := w.Flush(); err != nil {
+		log.Fatal(err)
```

Please return the error

---

Done